lms/djangoapps/courseware/management/commands/dump_to_neo4j.py

```diff
@@ -139,7 +139,21 @@ def coerce_types(value):
 class Command(BaseCommand):
     """
     Command to dump modulestore data to neo4j
+
+    Takes the following positional arguments:
+      host: the host of the neo4j server
+      port: the port on the server that accepts https requests
+      user: the username for the neo4j user
+      password: the user's password
+
+    Example usage:
+      python manage.py lms dump_to_neo4j host 7473 user password --settings=aws
     """
+    def add_arguments(self, parser):
+        parser.add_argument('host', type=unicode)
```

Comment:
I'd prefer explicit rather than positional arguments, but I'll call that a nit.

I do think you should collapse host and port to url however and obviate the code the smash to together hosts and ports.